lib/rubocop/cop/style/multiple_comparison.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks against comparing a variable with multiple items,
+      # where `Array.include?` could be used instead to avoid DRY errors.
```

Comment:
DRY errors? I'd simply use the term "code repetition" here. 

Technically speaking using `Set#include?` is even better, btw. Maybe this should be made configurable in respect to the message?

Array.include? -> Array#includes?